test/Npgsql.Tests/ReaderOldSchemaTests.cs

```diff
@@ -118,32 +118,18 @@ public async Task IsReadOnly()
 CREATE TABLE {table} (id SERIAL PRIMARY KEY, int2 SMALLINT);
 CREATE OR REPLACE VIEW {view} (id, int2) AS SELECT id, int2 + int2 AS int2 FROM {table}");
 
-        var command = new NpgsqlCommand($"SELECT * FROM {view}", conn);
+        var command = new NpgsqlCommand($"SELECT id, int2 FROM {view}", conn);
 
-        using var dr = command.ExecuteReader();
+        using var dr = command.ExecuteReader(CommandBehavior.SchemaOnly |CommandBehavior.KeyInfo);
```

Comment:
Possibly a good idea to mirror the tests from ReaderNewSchemaTests, which also test the case where KeyInfo isn't provided. Though since the "old" DataTable-based API simply relies on the "new" one and copies its values, that coverage isn't that important.

Reply:
Ideally, yes, though as you said `ReaderNewSchemaTests` should catch most of these problems.